shortfin/python/shortfin_apps/sd/components/tokenizer.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from pathlib import Path
+
+from transformers import CLIPTokenizer, BatchEncoding
```

Comment:
Yeah, we're going to want to see how to not need this. But whatever it takes for now.